Rubberduck.Resources/RubberduckUI.de.resx

```diff
@@ -1542,7 +1542,4 @@ Import abgebrochen.</value>
 
 Import abgebrochen.</value>
   </data>
-  <data name="SaveAndClose" xml:space="preserve">
-    <value>Speichern &amp; Schließen</value>
-  </data>
```

Comment:
Why have you deleted this?

Reply:
Resource may havr been changed with an earlier PR, but don't quote me on that

Reply:
The deletion is in the commit with the other deletions. This translation has only been just added by my translation PR. It really should not have been deleted.

Reply:
That's an incorrect automatic merge resolution then

Reply:
I tried reverting the commit and reapplying the changes. It again considers `SaveAndClose` to be a deletion.

Reply:
[It's here](https://github.com/rubberduck-vba/Rubberduck/pull/5263/files#diff-750e5fb8bdf9952a807fcab865e50c7aL1443)